monitor/core/src/main/java/core/JsonParsing.java

```diff
@@ -0,0 +1,131 @@
+package core;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+/**
+ * 
+ * @author Dionysis Athanasopoulos <dionysiscsuoi@gmail.com>
+ *
+ */
+
+public class JsonParsing {
+
+	private static void getArray( Object object ) throws ParseException {
```

Comment:
there are tons of json parser out there, gson, jackson just to name the most famous. Do we really want to maintain this custom code?